app/src/main/java/com/movies/presentation/home/custom_view/NavigationView.kt

```diff
@@ -17,45 +17,45 @@ class NavigationView @JvmOverloads constructor(
         NavigationCustomViewBinding.inflate(LayoutInflater.from(context), this, true)
 
     init {
-        setButtonsActiveStatus(NavigationButtons.HOME)
+        setButtonsActiveStatus(NavigationButtons.LEFT_BUTTON)
         setDefaultButtonTypes()
     }
 
-    fun homeButtonListener(callback: () -> Unit) {
+    fun firstButtonListener(callback: () -> Unit) {
         with(binding) {
-            navigationView.setOnClickListener {
+            leftButton.setOnClickListener {
                 callback()
-                setButtonsActiveStatus(NavigationButtons.HOME)
+                setButtonsActiveStatus(NavigationButtons.LEFT_BUTTON)
             }
         }
     }
 
-    fun favouritesButtonListener(callback: () -> Unit) {
+    fun secondButtonListener(callback: () -> Unit) {
```

Comment:
if you name buttons left and right, it will be better to name rightButtonListener

Reply:
done